packages/jaeger-ui/src/components/Monitor/ServicesView/__snapshots__/index.test.js.snap

```diff
@@ -79,11 +79,23 @@ exports[`<MonitorATMServicesView> ATM snapshot test 1`] = `
               "className": "select-a-timeframe-input",
               "clearable": false,
               "defaultValue": Object {
-                "label": "Last Hour",
-                "value": 3600000,
+                "label": "Last 5 minutes",
+                "value": 300000,
```

Comment:
What's the reason for changing the defaults in tests?

Reply:
my bad, is fixed now.